ros_buildfarm/ci_job.py

```diff
@@ -182,16 +182,12 @@ def configure_ci_job(
             'choose one of the following: %s' % ', '.join(sorted(
                 build_file.targets[os_name][os_code_name])))
 
-    if len(build_file.underlay_from_ci_jobs) > 1:
-        raise JobValidationError(
-            'Only a single underlay job is currently supported, but the ' +
-            'build file lists %d.' % len(build_file.underlay_from_ci_jobs))
-
-    underlay_source_job = None
+    underlay_source_jobs = None
```

Comment:
Same comment as on the other PR: I would remove the special case `None` here and just make it an empty list.